akd_client/src/ecvrf.rs

```diff
@@ -165,8 +164,9 @@ impl VRFPublicKey {
         self.verify(&proof, &message)?;
 
         let output: Output = (&proof).into();
-        let mut expected_truncated_hash: [u8; 32] = [0u8; 32];
-        expected_truncated_hash.copy_from_slice(&output.to_bytes()[..32]);
+        // truncate the 64-byte hash to first 32 bytes
```

Comment:
I think this was discussed in the chat but let's add a comment here why this is okay.